docs/synapse/userguides/ug015_storm_ref_pivot.rst

```diff
@@ -0,0 +1,41 @@
+Storm Reference - Pivot Operators
+=================================
+
+The operators below can be used to pivot from one set of nodes to another set of nodes within a Synapse hypergraph based on a set of specified criteria.
+
+All of these operators are defined in `storm.py`__.
+
+See the `Storm Syntax Conventions`__ for an explanation of the usage format used below.
+
+Where specific query examples are given, they are commonly provided in pairs using operator syntax followed by the equivalent macro syntax (if available).
+
+pivot()
+-------
+Todo
+
+join()
+------
+Todo
+
+refs()
+------
+Todo
+
+fromtags()
+----------
+Todo
+
+totags()
+--------
+Todo
+
+jointags()
+----------
+Todo
+
+
+.. _storm.py: ../../../synapse/lib/storm.py
+__ storm.py_
+
+.. _conventions: ../userguides/ug011_storm_basics.rst#syntax-conventions
```

Comment:
If we were to make all of these links to html, does that break our ability to generate linked PDFs?